DTS-STN/Service-Canada-Labs
# Create unit test for fragment render component

# [Create unit test for fragment render component](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities?workitem=162910)

## Test Instructions

1. yarn storybook 
2. verify FragmentRender renders components
3. yarn test:unit 
4. verify all tests pass
